model/clusters_mgmt/v1/sts_type.model

```diff
@@ -45,4 +45,7 @@ struct STS {
 
 	// Private key used to sign the service account token and allow cluster resources to authenticate against the cloud provider
 	BoundServiceAccountSigningKey String
+
+	// Kms Id for the bound service account signing key
```

Comment:
The comment needs a '.' for swagger to properly generate documentation (also in line 46)